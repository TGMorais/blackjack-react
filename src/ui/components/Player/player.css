.player  {
    font-family: 'Dosis', 'sans serif';
    color: white;
    margin: .5rem;
    font-size: 2rem;
}