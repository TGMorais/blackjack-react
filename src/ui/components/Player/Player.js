import React from "react";
import Hand from '../Hand/Hand';

import './player.css';

const Player = ({ hand = [], className }) => {
  return (
      <div className={`player ${className}`}>
          <Hand cards={hand} />
      </div>
  )
};

export default Player;
