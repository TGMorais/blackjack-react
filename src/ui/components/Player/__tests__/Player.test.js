import React from "react";
import ReactDOM from "react-dom";
import Player from "../Player";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Player />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders correctly", () => {
  const tree = renderer.create(<Player>Button</Player>).toJSON();
  expect(tree).toMatchSnapshot();
});