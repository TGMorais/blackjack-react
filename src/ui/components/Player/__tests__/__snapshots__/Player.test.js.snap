// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="player undefined"
>
  <div
    className="hand flex"
  />
</div>
`;
